site/content/installation-upgrade/installation-plus.md

```diff
@@ -380,7 +380,55 @@ Before you install NGINX Agent for the first time on your system, you need to se
     ```
     [nginx-agent]
     name=nginx-agent repo
-    baseurl=https://pkgs.nginx.com/nginx-agent/amzn2/$releasever/$basearch
+    baseurl=https://packages.nginx.org/nginx-agent/amzn/2023/$basearch/
+    sslclientcert=/etc/ssl/nginx/nginx-repo.crt
+    sslclientkey=/etc/ssl/nginx/nginx-repo.key
+    gpgcheck=0
+    enabled=1
+    ```
+
+1. To install `nginx-agent`, run the following command:
+
+    ```shell
+    sudo yum install nginx-agent
+    ```
+
+1. When prompted to accept the GPG key, verify that the fingerprint matches `573B FD6B 3D8F BC64 1079 A6AB ABF5 BD82 7BD9 BF62`, and if so, accept it.
+
+1. Verify the installation:
+
+    ```shell
+    sudo nginx-agent -v
+    ```
+
+### Installing NGINX Agent on Amazon Linux 2
+
+1. Create the `/etc/ssl/nginx` directory:
+
+    ```shell
+    sudo mkdir -p /etc/ssl/nginx
+    ```
+
+1. Log in to [MyF5 Customer Portal](https://account.f5.com/myf5/) and download your `nginx-repo.crt` and `nginx-repo.key` files.
+
+1. Copy the `nginx-repo.crt` and `nginx-repo.key` files to the `/etc/ssl/nginx/` directory:
+
+    ```shell
+    sudo cp nginx-repo.crt nginx-repo.key /etc/ssl/nginx/
+    ```
+
+1. Install the prerequisites:
+
+    ```shell
+    sudo yum install yum-utils procps ca-certificates
```

Comment:
sudo dnf install procps-ng ca-certificates (not sure about yum-utils and the exact name of procps)